src/site/content/en/blog/introducing-aurora/index.md

```diff
@@ -0,0 +1,225 @@
+readiness. Instead of asking every developer and site owner to become an expert
+in these areas and keep up with the constantly changing best practices, the
+framework can support these with baked-in solutions. This empowers developers
+and enables them to focus on building product features.
+
+In a nutshell, our vision is that a high bar of UX quality becomes a side effect
+of building for the web.
+
+## Aurora: a collaboration between Chrome and open-source web frameworks & tools
+
+For almost two years, we have worked with some of the most popular frameworks
+such as Next.js, Nuxt.js and Angular, working to improve web performance. We've
+also funded popular tools and libraries such as Vue, Eslint, Webpack. Today, we
+are giving this effort a name - **Aurora**.
+
+An aurora is a natural light display that shimmers in the sky. As we are trying
+to help user experiences built with frameworks shimmer and shine, we thought
+this name was an appropriate choice.
+
+{% Img src="image/0SXGYLkliuPQY3aSy3zWvdv7RqG2/lxaUR7Za1DoQMJuJEZHP.png",
+alt="Aurora logo", width="800", height="450" %}
+
+In the coming months, we'll be sharing a lot more detail on Aurora. This is a
+collaboration between a small team of Chrome engineers (internally codenamed
+WebSDK) with experience working on front-end stacks, and framework and tooling
+authors. Our goal is to deliver the best user-experience possible for production
+apps regardless of the browser you're rendering in.
+
+## What is our strategy?
+
+At Google, we've learned a lot while using frameworks and tools to build and
+maintain large scale web applications such as Google Search, Maps, Image Search,
+Google Photos etc. We discovered how frameworks can play a crucial role in
+predictable app quality by providing **strong defaults** and **opinionated
+tooling**.
+
+Frameworks have a unique vantage point for influencing both DX and UX as they
+span the entire system: the client and the server, the development and
+production environments, and they integrate tools such as compiler, bundler,
+linter etc.
+
+<figure class="w-figure">
+  {% Img src="image/0SXGYLkliuPQY3aSy3zWvdv7RqG2/UuOwHlcV76N8BwboHBVy.png", alt="Chart that shows
+  common tooling in frameworks", width="800", height="450" %}
+  <figcaption class="w-figcaption">
+    Common tooling used by framework developers
+  </figcaption>
+</figure>
+
+When solutions are baked into the framework, teams of developers can use these
+solutions and focus their time on what matters most to the product -- shipping
+great features for users.
+
+While we work to improve tools that live in every layer of the stack, frameworks
+such as Next.js, Nuxt.js, and Angular CLI, manage every step of an application's
+lifecycle. For this reason, and the fact that React adoption is the largest
+within the core UI framework ecosystem, most of our optimizations have begun as
+a proof of concept within Next.js before expanding to the rest of the ecosystem.
```

Comment:
"begun with proving out in Next.js" 
(being more direct: we are shipping production features here and not just doing proof of concept work)

Reply:
Updated